config/galaxy.ini.sample

```diff
@@ -212,6 +212,11 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # Set to True to instruct Galaxy to install Conda from the web automatically
 # if it cannot find a local copy and conda_exec is not configured.
 #conda_auto_init = False
+# Set to True if conda_prefix and job_working_directory are not on the same
+# volume. Conda will copy packages content instead of creating symlinks.
```

Comment:
the code for this is there: https://github.com/conda/conda/blob/8257927e0061f537f77a6999e971a33331dec941/conda/misc.py#L157